### ISSUE TITLE
`chocolatey_package` timeout should default to the Chocolatey `commandExecutionTimeoutSeconds` config setting

### ISSUE THREAD

**Issue by bender-the-greatest** [chef/chef#8733]
### Describe the Enhancement:
The `chocolatey_package` resource does not honor the `commandExecutionTimeoutSeconds` choco config setting, instead relying on a Chef-internal 900 second timeout by default.

### Describe the Need:
Especially because we can set the client options with the `chocolatey_config` resource, including the `commandExecutionTimeoutSeconds` setting, it becomes confusing when trying to troubleshoot why your package install suddenly bombs out at 900 seconds when the default choco timeout is 2700 seconds, and recommended to be 14400 for enterprises.

### Current Alternative
Set `timeout` on all `chocolatey_package` resources. This seems redundant though when we have set a client default.

### Can We Help You Implement This?:
If this is met positively I would love to take a crack at implementing this.

**Comment by lamont-granquist** [comment]
yeah you can give that a shot, the way it picks up the timeout value from shell_out is a bit magical, and you need to not affect any global behavior and keep the changes isolated to the chocolatey package resource.
